### ISSUE TITLE
feature: redis extension

### ISSUE THREAD

**Issue by vvtll** [peachpiecompiler/peachpie#1016]
can we implement the php redis extension in peachpie?

**Comment by menturion** [comment]
Did you try to use the vendor lib Predis? Predis is a PHP implementation that "does not require any additional C extension by default".

See...
- https://packagist.org/packages/predis/predis
- https://github.com/predis/predis

**Comment by vvtll** [comment]
> Did you try to use the vendor lib Predis? Predis is a PHP implementation that "does not require any additional C extension by default".
> 
> See...
> 
> * https://packagist.org/packages/predis/predis
> * https://github.com/predis/predis

Yes,I try.But the Predis use a lot of _destruct function,now Peachpie can not support that.

**Comment by menturion** [comment]
I see.

It is a pity that such a popular library with 111 392 852 installs still does not work on PeachPie.

**Comment by bfistein** [comment]
We try to implement stuff as needed by our clients or by the community. So far nobody requested this, so we are noting it ;)

**Comment by jakubmisek** [comment]
@vvtll is the support for __destruct() needed for proper functionality?

is there anything else not working we can do in PeachPie?

**Comment by avriltank** [comment]
nothing.predis  just  use  sream_socket  to  use 
redis ,that  works  fine  in  peachpie
